<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSP Process Explorer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="qlsSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="SessionTree" name="qtvSessions">
       <property name="headerHidden">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="qlvExpression" stretch="0,0">
        <item>
         <layout class="QHBoxLayout" name="qlhExpressionEntry">
          <item>
           <widget class="QLabel" name="qlbExpression">
            <property name="text">
             <string>Expression:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="qleExpression"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="ProcessTree" name="qtvExplorer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="headerHidden">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="qlvTrace_2">
        <item>
         <layout class="QHBoxLayout" name="qlhTraceLabel">
          <item>
           <widget class="QLabel" name="qlbTrace">
            <property name="text">
             <string>Trace:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="qleReference">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>0</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="TraceListWidget" name="qlvTrace">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="qmbMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>836</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="qmmFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="qmaOpen"/>
   </widget>
   <addaction name="qmmFile"/>
  </widget>
  <widget class="QStatusBar" name="qsbStatus"/>
  <action name="qmaOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ProcessTree</class>
   <extends>QTreeView</extends>
   <header>widget/processtree.h</header>
   <slots>
    <signal>itemSelected(QModelIndex)</signal>
    <slot>loadInitialState()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>TraceListWidget</class>
   <extends>QListView</extends>
   <header>widget/tracelistwidget.h</header>
   <slots>
    <slot>setTraces(QModelIndex)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>SessionTree</class>
   <extends>QTreeView</extends>
   <header>widget/sessiontree.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qleExpression</sender>
   <signal>returnPressed()</signal>
   <receiver>qtvExplorer</receiver>
   <slot>loadInitialState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qmaOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>actionOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qtvExplorer</sender>
   <signal>itemSelected(QModelIndex)</signal>
   <receiver>qlvTrace</receiver>
   <slot>setTraces(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>actionOpen()</slot>
 </slots>
</ui>
